### PR TITLE
test/test: check access

### PR DESCRIPTION
**Pull request by 31Demigod31**
check access

### CODE REVIEW

**Review by 31Demigod31 — COMMENTED** [review]
resolved comments

File: frontend/src/pages/Movies/index.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ class Movies extends Component {
                 type="number"
                 iconClass="fas fa-star"
               />
-              { loggedIn && <Link to='/movies/new' className='btn btn-primary btn-block my-2 text-white'> Add Product </Link> }
+              { loggedIn && <Link to='/movies/new' className='btn btn-primary btn-block my-2 text-white'> Add Product test</Link> }
```
Comment (hamellton) [inline frontend/src/pages/Movies/index.js:85]: remove it

Comment (hamellton) [reply frontend/src/pages/Movies/index.js:85]: resolve

**Comment by hamellton** [comment]
merge

**Review by hamellton — APPROVED** [review]

**Pull request merged**